src/Microsoft.PowerShell.Commands.Utility/commands/utility/Get-Error.cs

```diff
@@ -11,7 +11,6 @@ namespace Microsoft.PowerShell.Commands
     /// <summary>
```

Comment:
@SteveL-MSFT, your last commit had 5 failures in `PowerShell-CI-macos`
Error occurred in test script 'test/powershell/Modules/Microsoft.PowerShell.Utility/Foreach-Object-Parallel.Tests.ps1'
```powershell
At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Foreach-Object-Parallel.Tests.ps1:106 char:1
+ }
+ ~
Unexpected token '}' in expression or statement.
at <ScriptBlock>, /Users/runner/runners/2.160.1/work/1/a/bins/publish/Modules/Pester/4.8.0/Pester.psm1: line 1097
```
Error occurred in test script '/Users/runner/runners/2.160.1/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Foreach-Object-Parallel.Tests.ps1'
```powershell
At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Foreach-Object-Parallel.Tests.ps1:106 char:1
+ }
+ ~
Unexpected token '}' in expression or statement.
at <ScriptBlock>, /Users/runner/runners/2.160.1/work/1/a/bins/publish/Modules/Pester/4.8.0/Pester.psm1: line 1097
```
Error occurred in test script '/Users/runner/runners/2.160.1/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Foreach-Object-Parallel.Tests.ps1'
```powershell
At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Foreach-Object-Parallel.Tests.ps1:106 char:1
+ }
+ ~
Unexpected token '}' in expression or statement.
at <ScriptBlock>, /Users/runner/runners/2.160.1/work/1/a/bins/publish/Modules/Pester/4.8.0/Pester.psm1: line 1097
```
Error occurred in test script '/Users/runner/runners/2.160.1/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Foreach-Object-Parallel.Tests.ps1'
```powershell
At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Foreach-Object-Parallel.Tests.ps1:106 char:1
+ }
+ ~
Unexpected token '}' in expression or statement.
at <ScriptBlock>, /Users/runner/runners/2.160.1/work/1/a/bins/publish/Modules/Pester/4.8.0/Pester.psm1: line 1097
```
Error occurred in test script '/Users/runner/runners/2.160.1/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Foreach-Object-Parallel.Tests.ps1'
```powershell
At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Foreach-Object-Parallel.Tests.ps1:106 char:1
+ }
+ ~
Unexpected token '}' in expression or statement.
at <ScriptBlock>, /Users/runner/runners/2.160.1/work/1/a/bins/publish/Modules/Pester/4.8.0/Pester.psm1: line 1097
```